testsuite/features/core/allcli_sanity.feature

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2019 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Sanity checks
+  In order to use the product
+  I want to be sure to use a sane environment
+
+  Scenario: The server is healthy
+    Then "server" should have a FQDN
+
+  Scenario: The traditional client is healthy
+    Then "sle-client" should have a FQDN
+    And "sle-client" should communicate with the server
+
+  Scenario: The minion is healthy
+    Then "sle-minion" should have a FQDN
+    And "sle-minion" should communicate with the server
+
+@ssh_minion
+  Scenario: The SSH minion is healthy
+    Then "ssh-minion" should have a FQDN
+    And "ssh-minion" should communicate with the server
+
+@proxy
+  Scenario: The proxy is healthy
+    Then "proxy" should have a FQDN
+    And "proxy" should communicate with the server
+
+@centos_minion
+  Scenario: The Centos minion is healthy
+    Then "ceos-minion" should have a FQDN
+    And "ceos-minion" should communicate with the server
+
+@ubuntu_minion
+  Scenario: The Ubuntu minion is healthy
+    Then "ubuntu-minion" should have a FQDN
+    And "ubuntu-minion" should communicate with the server
+
+@virthost_kvm
+  Scenario: The KVM host is healthy
+    Then "kvm-server" should have a FQDN
+    And "kvm-server" should communicate with the server
+
+@virthost_xen
+  Scenario: The Xen host is healthy
+    Then "xen-server" should have a FQDN
+    And "xen-server" should communicate with the server
+
+  Scenario: The external repositories can be reached
```

Comment:
Shouldn't we also include check for github.com too? Something like minimal checkout of spacewalk repository. I remember problems we had on that during spring.

Reply:
Atm it will be late, we have github issues on deployment stage, later we will not use it.

Reply:
Well, not in this PR for sure :-D

Reply:
It was my idea too, to check github. But the only calls we have for it are:
```
features/step_definitions/docker_steps.rb:  cont_op.create_store('fake_store', 'https://github.com/SUSE/spacewalk-testsuite-base', 'registry')
features/step_definitions/docker_steps.rb:  cont_op.create_store('Norimberga', 'https://github.com/SUSE/spacewalk-testsuite-base', 'registry')
```
And they connect to repo `https://github.com/SUSE/spacewalk-testsuite-base` that has been deprecated for years Besides, these are fakes. I am not sure what to do here.

Guidance welcome.

Reply:
I did something. Not sure we're even using github at all.